media/js/devreg/l10n.js

```diff
@@ -225,11 +225,12 @@ $(document).ready(function () {
         $("#change-locale").text(current.text());
         $(".trans").each(function () {
             var $el = $(this),
+                $ni = null,
```

Comment:
not keen on making changes beyond what we need for jshint, otherwise the number of changes becomes too big and scary